<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>docpropertiesdialog</class>
 <widget class="QWidget" name="docpropertiesdialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>472</width>
    <height>178</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Document Properties</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <widget class="QTabWidget" name="DocumentTabWidget">
     <property name="currentIndex">
      <number>3</number>
     </property>
     <widget class="QWidget" name="wordInfoTab">
      <attribute name="title">
       <string>&amp;Word Info</string>
      </attribute>
      <layout class="QFormLayout" name="formLayout">
       <item row="0" column="0">
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <property name="sizeConstraint">
          <enum>QLayout::SetDefaultConstraint</enum>
         </property>
         <item>
          <widget class="QLabel" name="wordCountLabel">
           <property name="text">
            <string>Word Count:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="wordComboBox">
           <item>
            <property name="text">
             <string>All Words</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Unique Words</string>
            </property>
           </item>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="wordResultLabel">
           <property name="text">
            <string>0</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="charactInfoTab2">
      <attribute name="title">
       <string>&amp;Character Info</string>
      </attribute>
      <layout class="QHBoxLayout" name="horizontalLayout_4">
       <item>
        <widget class="QLabel" name="charcountLabel">
         <property name="text">
          <string>Character Count</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="characterComboBox">
         <item>
          <property name="text">
           <string>Total Characters (excluding spaces)</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Total Characters (including spaces)</string>
          </property>
         </item>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="charResultLabel">
         <property name="text">
          <string>0</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="miscellaneousTab3">
      <attribute name="title">
       <string>&amp;Miscellaneous</string>
      </attribute>
      <layout class="QHBoxLayout" name="horizontalLayout_7">
       <item>
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_6">
           <property name="spacing">
            <number>6</number>
           </property>
           <property name="bottomMargin">
            <number>0</number>
           </property>
           <item>
            <widget class="QLabel" name="paragraphCountLabel">
             <property name="text">
              <string>Paragraph Count:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="paraResultLabel">
             <property name="text">
              <string>0</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_5">
           <item>
            <widget class="QLabel" name="LineCountLabel">
             <property name="text">
              <string>Line count:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="linecountResultLabel">
             <property name="text">
              <string>0</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="graphTab4">
      <attribute name="title">
       <string>&amp;Graph</string>
      </attribute>
      <widget class="QPushButton" name="okButton">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>110</y>
         <width>80</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>OK</string>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
      </widget>
      <widget class="QPushButton" name="cancelButton">
       <property name="geometry">
        <rect>
         <x>120</x>
         <y>110</y>
         <width>80</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>Cancel</string>
       </property>
      </widget>
      <widget class="QWidget" name="layoutWidget">
       <property name="geometry">
        <rect>
         <x>12</x>
         <y>42</y>
         <width>170</width>
         <height>24</height>
        </rect>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout_9">
        <item>
         <widget class="QLabel" name="dataSourceLabel">
          <property name="text">
           <string>Data source:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="dataSourceComboBox">
          <item>
           <property name="text">
            <string>File</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Auto Generate</string>
           </property>
          </item>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="layoutWidget">
       <property name="geometry">
        <rect>
         <x>12</x>
         <y>10</y>
         <width>114</width>
         <height>24</height>
        </rect>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout_8">
        <item>
         <widget class="QLabel" name="chartTypeLabel">
          <property name="text">
           <string>Chart Type:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="chartTypComboBox">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <item>
           <property name="text">
            <string>Line</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Bar</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Pie</string>
           </property>
          </item>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="layoutWidget">
       <property name="geometry">
        <rect>
         <x>12</x>
         <y>71</y>
         <width>95</width>
         <height>24</height>
        </rect>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout_10">
        <item>
         <widget class="QLabel" name="quantityLabel">
          <property name="text">
           <string>Quantity:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="quantityComboBox">
          <item>
           <property name="text">
            <string>10</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>20</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>All</string>
           </property>
          </item>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
